test/numa/tests/memtier/n4c16/topology.var.json

```diff
@@ -0,0 +1,5 @@
+[
+    {"mem": "1G", "cores": 2, "nodes": 2, "packages": 2},
+    {"nvmem": "8G", "node-dist": {"5": 28, "0": 17}},
```

Comment:
I wouldn't include nvmem/PMEM in the basic test set, since we have neither positive nor negative checks for them. We'll probably create a dedicated test set for it later.

Reply:
Ok, I'll change topology to more basic one.

On one hand, topologies are very expensive as they require their own VMs. On the other hand, it's the separate topologies that enable cleanest cut to run tests in parallel at some point...

Reply:
True, we should prevent them from mushrooming too much.